x-pack/plugin/ilm/qa/with-security/src/test/java/org/elasticsearch/xpack/security/PermissionsIT.java

```diff
@@ -126,7 +133,46 @@ public void testCanViewExplainOnUnmanagedIndex() throws Exception {
         assertOK(client().performRequest(request));
     }
 
-    private void createNewSingletonPolicy(String policy, String phaseName, LifecycleAction action) throws IOException {
+    public void testILMPolicyWithRolloverActionWhereUserRoleIsLimitedByAliasOfIndex() throws IOException, InterruptedException {
+        createNewSingletonPolicy(adminClient(), "foo-policy", "hot", new RolloverAction(null, null, 2L));
```

Comment:
Actually, I have a question.

ILM executes its actions as the user that created the policy. In this case, it is the admin user, not `test_user`. 

This test is interested in verifying that the `test_user` has permissions to roll over the index? Looks like it may have permission to roll-over in this case, but it would not have permissions for other index-level actions, right?